TestProjects/iOS-Support/XamarinSidebar/MvvmCross.iOS.Support.Sidebar/Views/CenterPanelView.cs

```diff
@@ -42,7 +42,7 @@ public override void ViewDidLoad()
             var bindingSet = this.CreateBindingSet<CenterPanelView, CenterPanelViewModel>();
             bindingSet.Bind(label).To(vm => vm.ExampleValue);
             bindingSet.Bind(rightPanelInstructions).To(vm => vm.RightPanelInstructions);
-            bindingSet.Bind(masterButton).To(vm => vm.ShowMasterCommand);
+			bindingSet.Bind(masterButton).For("TouchDown").To(vm => vm.ShowMasterCommand);
```

Comment:
Can we still have TouchUpInside as default if nothing specificed, such that you don't break all Apps.